packages/web-react/src/components/Grid/demo/GridAlignment.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import DocsBox from '../../../../docs/DocsBox';
+import Grid from '../Grid';
+
+const GridAlignment = () => (
+  <>
+    <Grid cols={2}>
+      <Grid cols={1} UNSAFE_className="bg-cover">
+        <DocsBox size="small">
+          Stretch (normal)
```

Comment:
```suggestion
          Stretch
```

In CSS, there is literally the `stretch` option, not `normal`.

Reply:
oh, there should be "default"